cfgmgr/vlanmgr.cpp

```diff
@@ -282,6 +302,7 @@ void VlanMgr::doVlanTask(Consumer &consumer)
             if (isVlanStateOk(key) && m_vlans.find(key) == m_vlans.end())
             {
                 m_vlans.insert(key);
+                m_vlanReplay.erase(kfvKey(t));
```

Comment:
>kfvKey(t) [](start = 35, length = 9)

erase `it` directly?

Reply:
@qiluo-msft sorry, I didn't understand, here I am erasing the key from the set m_vlanReplay, what do you mean by erase 'it' directly?